### DISCUSSION TITLE
Default initramfs in Archlinux

### DISCUSSION THREAD

**Discussion by ht990332** [anatol/booster#130] [General]
Do you think booster could eventually replace mkinitcpio as the default image creation utility in Arch Linux? Or is that no the intention?
I admire booster because it doesn't have "hooks" like mkinitcpio does and this means that other packages can stay closer to vanilla/upstream. I prefer centralization and consolidation to fragmentation, partially why I like systemd as well.

There are obvious bugs in booster that should be fixed first such as a hardware race in Booster that I read about earlier when searching for TPM related issues and maybe adding plymouth support. Still would you say this is viable goal?

**Comment by anatol** [comment]
Thanks for the kind words to `booster`. Seeing people find it useful is something important to me.

Whether booster become default initramfs option or not is more Arch-specific question and it is better to ask [broader Arch Linux community](https://lists.archlinux.org/listinfo/arch-general). At this point there are 3 implementations of initramfs generator in the Arch repos - booster, mkinitcpio and dracut. And different people prefer different tools due to features they are interested in.

One way to make booster more popular is to spread the word, improve documentation/[wiki page](https://wiki.archlinux.org/title/Booster), add tutorials.
